source/expectations.rst

```diff
@@ -78,43 +87,31 @@ other projects as they become available.
 Data Management
 ---------------
 
-When data is used by the lab, it is expected to reside on the shared
-filesystem.  Accompanying this should be an index or README file describing
-what the data *is*, where it originated from, and what restrictions there are
-on its use.  When data is stored on the shared filesystem, there is no
-expectation of privacy within the lab unless it is explicitly enforced; for
-special cases this can be dealt with differently.  Lab members are not
-encouraged to browse other directories, but it is not forbidden.  For example,
-this might be an example directory structure:::
-
-  /dpool/mturk/DarkSkySims-DS14a/README
-  /dpool/mturk/DarkSkySims-DS14a/ds14_a_1.0000.sdf
-  /dpool/mturk/LIDAR-Philly/README
-  /dpool/mturk/LIDAR-Philly/data*
-  /dpool/mturk/HRRR-20140726/README
-  /dpool/mturk/HRRR-20140726/dir1/*
-  /dpool/mturk/HRRR-20140726/dir2/*
-
-and so on.  In the near-term future, we will be deploying a data cataloging
-system which will be fed with the data from README files, and which will be the
-primary cataloging system in the future.
-
-Wherever possible, names should conform to including either a date (if the data
-is tied to a specific time, parameter query or observation) or a
-semantically-meaningful name (if it is part of a larger project.)
+We utilize `git-annex` for internal data sharing, and for sharing data
```

Comment:
Here you're talking about using git annex for data sharing and you've removed some of the documentation about storing data in /dpool/. In `infrastructure.rst` you mention dpool as a computing resource for file storage. Maybe add in information about how you expect these to interoperate?

Reply:
Whoops, gotta get rid of the dpool stuff.